src/Sylius/Component/Core/Model/OrderItemUnit.php

```diff
@@ -135,4 +135,20 @@ public function setShippingState($state)
     {
         $this->shippingState = $state;
     }
+
+    /**
+     * Returns sum of neutral and non neutral tax adjustments.
+     *
+     * {@inheritdoc}
+     */
+    public function getTaxTotal()
```

Comment:
This is good idea, but maybe we could introduce this in separate PR, to not hold this one longer and to make review simpler? This could also give us possibility to discuss about such an approach in more comfortable circumstances ;)